parser/src/ast.rs

```diff
@@ -391,3 +391,28 @@ pub enum StringGroup {
         values: Vec<StringGroup>,
     },
 }
+
+#[derive(Debug, PartialEq)]
+pub enum Varargs {
+    None,
+    NoCapture,
```

Comment:
Suggestion to rename this into UnNamed and Named.

Reply:
Sounds good.